src/Core/Services/VaultTimeoutService.cs

```diff
@@ -69,7 +76,7 @@ public async Task<bool> IsLockedAsync(string userId = null)
             {
                 if (await _cryptoService.HasAutoUnlockKeyAsync(userId))
                 {
-                    await _cryptoService.SetUserKeyAsync(await _cryptoService.GetAutoUnlockKeyAsync(userId));
+                    await _cryptoService.SetUserKeyAsync(await _cryptoService.GetAutoUnlockKeyAsync(userId), userId);
                 }
                 else
                 {
```

Comment:
🎨 To avoid the else (Github doesn't let me grab the other part which returns), put the if the other way around:
```c#
if (!await _cryptoService.HasAutoUnlockKeyAsync(userId))
{
    return true;
}

await _cryptoService.SetUserKeyAsync(await _cryptoService.GetAutoUnlockKeyAsync(userId));
await _cryptoService.SetUserKeyAsync(await _cryptoService.GetAutoUnlockKeyAsync(userId), userId);
```